### ISSUE TITLE
the Loss not converging

### ISSUE THREAD

**Issue by gugugilin** [xuannianz/keras-CenterNet#5]
Has anyone have happened this case，my loss always in 5 range，while I testing my image
we got too low score and not correct bounding-box

Which the range of this Loss is fine ?

**Comment by xuannianz** [comment]
Hi @gugugilin, loss can be affected by a lot of factors. Even modification of a single hyper parameters may lead to losses of different magnitudes. So "fine loss range" is unknowable.
I prefer to monitor the mAP of validation set during training.
Here are some tips when train this project:
1. Check the generator. Make sure correct data is feed to the training model.
2. If the loss does not converge at the early steps,reduce the lr.
3. CenterNet seems to converge slowly.So please by more patient.

**Comment by gugugilin** [comment]
> Hi @gugugilin, loss can be affected by a lot of factors. Even modification of a single hyper parameters may lead to losses of different magnitudes. So "fine loss range" is unknowable.
> I prefer to monitor the mAP of validation set during training.
> Here are some tips when train this project:
> 
> 1. Check the generator. Make sure correct data is feed to the training model.
> 2. If the loss does not converge at the early steps,reduce the lr.
> 3. CenterNet seems to converge slowly.So please by more patient.

Thanks for your replay, I have some question ，if you don't mind

I didn't overwrite the loss function and use the other backbone of a pre-trained model from Imagenet, Why they cause the range of loss has a huge difference between MobileNetv2 and ResNet

FYI, I am trying to reduce the learning rate， hope it is helpful for my case

**Comment by gugugilin** [comment]
I found that the focal loss which let my training become wirde，because my dataset has too much negative sample，so they always predict to zero, I update the parament of focal loss, it seems fine as far
